dexbot/strategies/base.py

```diff
@@ -153,9 +153,9 @@ def configure(cls, return_base_config=True):
             ConfigElement('account', 'string', '', 'Account',
                           'BitShares account name for the bot to operate with',
                           ''),
-            ConfigElement('market', 'string', 'USD:BTS', 'Market',
-                          'BitShares market to operate on, in the format ASSET:OTHERASSET, for example \"USD:BTS\"',
-                          r'[A-Z0-9\.]+[:\/][A-Z0-9\.]+'),
+            ConfigElement('market', 'string', 'BTS/USD', 'Market',
+                          'BitShares market to operate on, in the format QUOTE/BASE, for example \"BTS/USD\"',
+                          r'[A-Z\.]+[:\/][A-Z\.]+'),
```

Comment:
You lost changes introduced in 9a0fc56b417fac1c4dbde43a65bdb941709a2a58, please rebase your changes to latest `devel` branch